File: Desenvolvimento/class/itens.php
<?php

class itens {

    private $cod_itens;
    private $nome_peca;
    private $largura;
    private $altura;
    private $qtde;

    public function setCod_itens($cod_itens){
        $this->cod_itens = $cod_itens;
    }
    public function getCod_itens(){
        return $this->cod_itens;
    }
    public function setNome_peca($nome_peca){
        $this->nome_peca = $nome_peca;
    }
    public function getNome_peca(){
        return $this->nome_peca;
    }
    public function setLargura($largura){
        $this->largura = $largura;
    }
    public function getLargura(){
        return $this->largura;
    }
    public function setAltura($altura){
        $this->altura = $altura;
    }
    public function getAltura(){
        return $this->altura;
    }
    public function setQtde($qtde){
        $this->qtde = $qtde;
    }
    public function getQtde(){
        return $this->qtde;
    }
}
?>

File: Sistema de Corte de Rochas - Caso de Uso/src/javafx/domain/Itens.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package javafx.domain;

/**
 *
 * @author André
 */
public class Itens {

    private int codItem;
    private String nomePeca;
    private float largura;
    private float altura;
    private int qtd;
    private PedidoDeCorte pedidoCorte;

    public PedidoDeCorte getPedidoCorte() {
        return pedidoCorte;
    }

    public void setPedidoCorte(PedidoDeCorte pedidoCorte) {
        this.pedidoCorte = pedidoCorte;
    }

    public int getCodItem() {
        return codItem;
    }

    public void setCodItem(int codItem) {
        this.codItem = codItem;
    }

    public String getNomePeca() {
        return nomePeca;
    }

    public void setNomePeca(String nomePeca) {
        this.nomePeca = nomePeca;
    }

    public float getLargura() {
        return largura;
    }

    public void setLargura(float largura) {
        this.largura = largura;
    }

    public float getAltura() {
        return altura;
    }

    public void setAltura(float altura) {
        this.altura = altura;
    }

    public int getQtd() {
        return qtd;
    }

    public void setQtd(int qtd) {
        this.qtd = qtd;
    }

}
